README.md

```diff
@@ -30,7 +30,7 @@ The following image object is expected by the component (when used as a *bower c
 Arguments are passed to the React component (when used as a *node package*) -
 
 ```javascript
-<NImage picClass={picClass} imgClass={imgClass} srcset={srcset} url={url} alt={alt}/>
+<NImage picClass={picClass} imgClass={imgClass} srcset={srcset} url={url} imgServiceReqd={imgServiceReqd} alt={alt}/>
```

Comment:
not sure `imgServiceReqd` is the best name. As a user I wouldn't have any idea what that means. don't have a better alternative though. maybe `isImageServiceUrl`?